dataStores/chat/ChatController.js

```diff
@@ -0,0 +1,32 @@
+const storage = require('../dataStores/Storage');
+
+class ChatController {
+
+	getMessages(req, res) {
+		return res.status(200).json({messages: storage.messages});
+	}
+
+	saveMessage(req, res) {
+		storage.messages.push(req.body.username);
+		return res.status(200).send();
+	}
+
+	connect(req, res) {
+		storage.users.push(req.body.username);
+		return res.status(200).send();
+	}
+
+	disconnect(req, res) {
+		const index = storage.users.indexOf(req.body.username);
+		if (index !== -1) {
+			storage.users.splice(index, 1);
+			return res.status(200).send();
```

Comment:
Po usunięciu jak nic nie zwracasz to 204 moim zdaniem, w momencie gdy przesyłasz jakąś zwrotkę np id kolejki czy coś takiego to wtedy 200 byłoby nawet wskazane.